packages/shared/src/components/layout/common.tsx

```diff
@@ -78,9 +81,21 @@ export const SearchControlHeader = ({
     iconOnly: true,
     buttonVariant: isLaptop ? ButtonVariant.Float : ButtonVariant.Tertiary,
   };
+
+  const feedsWithActions = [SharedFeedPage.MyFeed, SharedFeedPage.Custom];
+
   const actionButtons = [
-    feedName === SharedFeedPage.MyFeed ? (
-      <MyFeedHeading key="my-feed" onOpenFeedFilters={openFeedFilters} />
+    feedsWithActions.includes(feedName as SharedFeedPage) ? (
+      <MyFeedHeading
+        key="my-feed"
+        onOpenFeedFilters={() => {
+          if (feedName === SharedFeedPage.Custom && router.query?.slugOrId) {
+            router.push(`${webappUrl}feeds/${router.query.slugOrId}/edit`);
+          } else {
+            openFeedFilters();
```

Comment:
What the difference between this one and the one used in the empty screen?
Felt like they could do the same thing.

Reply:
They are not, but at this point I also don't see the smart way of extracting logic. Adding hook seems wasteful and component again won't do anything.

Reply:
Yeah no worries :)